projects/ngx-d3timeline/src/lib/options/options.ts

```diff
@@ -22,5 +27,8 @@ export const defaultOptions: CompleteOptions = {
   resourceAxis: {
     tickLineLength: 0,
     showGridLines: true
+  },
+  activities: {
+    lateralPadding: 0.25
```

Comment:
isn't this just resourcePadding? Not sure it should be part of activityOptions

Reply:
My thinking was that you could specify it at an activity(type) level as well as a global level. So Driving could be padded more than Handling etc. But now I don't think that is achievable via scale.paddingInner/Outer as there is only one scale. You had something else in your POC I think maxWidth but I didn't get my head around how the two interact, it seems like maybe they were serving a similar purpose. Probably best to have a chat about this one.

Reply:
moved to options.resource.gap